test/certificate.test.js

```diff
@@ -0,0 +1,25 @@
+require('./enviroment.test');
+
+const certificateController = require('../src/controllers/certificate.controller');
+const Certificate = require('../src/models/certificate.model');
+
+test('should return certificate details when valid certificate id is provided', async () => {
```

Comment:
Negative test missing.

I.E. should throw error when a non existing certificate id is provided.

Reply:
Negative test added